docs/installation.rst

```diff
@@ -98,6 +98,16 @@ On Ubuntu systems:
     export PATH=$PATH:~/.local/bin
     pip3 install ansible ansible-rulebook ansible-runner
 
+On macOS systems:
+
+.. code-block:: shell
+
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+    brew install openjdk@17
+    echo 'export PATH="/usr/local/opt/openjdk@17/bin:$PATH"' >> ~/.bash_profile
+    brew install python@3.x  # replace X with the version you want to install
```

Comment:
Maybe It's just me but I really like snippets that are fully functional by copy paste. I would prefer to leave the comment indicating that the python version can be replaced but I would add in the command python3.11 which is the official version supported in AAP. Same for pip